README.md

```diff
@@ -1 +1,84 @@
-# capstone2
\ No newline at end of file
+# Project Name 
+capstone: API-based webapp
+
+# Description of project.
+The JavaScript capstone project is about building  web application based on an external API. We selected an API that provides data about Meals and then built the webapp around it.
+
+## Project web demo link. 
+
+
+## Built With 
+- Html. 
+- Css. 
+- JS.
+- Webpark.
+- jest test
+
+Website link: 
+
+# Getting starting 
+To get a local copy up and running follow these simple example steps:
+
+## Prerequisites
+ Vscode. 
+ must have git installed.
+
+### Setup
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+[Download the ZIP from this location]( https://github.com/eerapheal/leader-board/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:
+
+```bash
+git clone  https://github.com/kennyegun24/capstone2.git
+- Once the files are on your machine, open the _todo-list_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up
+
+## 🛠 Installation & Set Up
+
+1. Install Node Modules
+
+   ```sh
+   npm install
+   
+   2. Start the development server
+
```

Comment:
- [ ] Your readme file should contains a link to the video with a clear and objective description of the project. Remember to highlight a peace of code you find interesting or you are proud of making.